### PR TITLE
Update docs (for python)

### PR DESCRIPTION
**Pull request by hmc-cs-mcmurphy**
I got rid of files that are now deprecated and updated the docstrings and documentation for all Python files. I did not work on anything other than python

### CODE REVIEW

File: teamworkApp/lib/dbCalls.py [hunk 1]
```diff
@@ -123,38 +122,43 @@ def get_all_student_IDs(test=False):
 
 def insert_student_team_pairs(student_and_team_ids, test=False):
     """
-    Insert list of student_id team_id pairs into database
+    Insert a list of (student_id, team_id pairs) into database
```
Comment (saramcallister) [inline teamworkApp/lib/dbCalls.py]: I would put pairs outside of the brackets if you leave the word in the comments.

File: teamworkApp/lib/generateTeams.py [hunk 2]
```diff
@@ -1,4 +1,8 @@
-import argparse 
+# muddersOnRails()
+#
```
Comment (saramcallister) [inline teamworkApp/lib/generateTeams.py:2]: Do we want a header on this file? @

File: teamworkApp/lib/importData.py [hunk 3]
```diff
@@ -134,9 +149,12 @@ def list_answer_data(student_dict):
     return answer_to_db
 
 def list_style_data(student_dict):
-	"""Given a dictionary of student data, calculate and import style data
-	for each student.
-	Uses find_style to calculate styles."""
+	"""
```
Comment (saramcallister) [inline teamworkApp/lib/importData.py]: The indentation on this looks funny

**Review by saramcallister — APPROVED** [review]
Thanks Maeve!!!!!!

File: teamworkApp/lib/generateTeams.py (hunk 2 above)
Comment (hmc-cs-mcmurphy) [reply teamworkApp/lib/generateTeams.py:2]: I didn't have enough information for this file to fill in the rest. Do you? or would Reagan or Nathan?

**Pull request merged**